tx-pool/src/block_assembler/mod.rs

```diff
@@ -549,20 +560,17 @@ impl BlockAssembler {
         current_epoch: &EpochExt,
         cellbase: TransactionView,
         entries: Vec<TxEntry>,
-    ) -> Result<Byte32, AnyError> {
+    ) -> Result<(Byte32, Vec<TxEntry>, usize), AnyError> {
```

Comment:
suggest to return `Result<(Byte32, Vec<TxEntry>)`, the total tx size can be calculated by `tx_entries.iter().map(entry.size).sum()` if needed